polyply/src/generate_templates.py

```diff
@@ -94,7 +94,7 @@ def find_interaction_involving(block, current_node, prev_node):
                  linking atom {} and atom {}.'''
         raise IOError(msg.format(block.nodes[0]["resname"], prev_node, current_node))
 
-def _expand_inital_coords(block, bond=None, pos=None, fixed=None,
+def _expand_inital_coords(block, max_count=50000, bond=None, pos=None, fixed=None,
```

Comment:
Docstring outdated

Reply:
I updated the docstring. Also since this is no longer just a wrapper around the Kamda-Kawai expansion I removed the other arguments which are not even propagated.